static/js/toc.js

```diff
@@ -131,10 +131,10 @@ const tableOfContents = {
   },
 
   getParam: (sname) => {
-    let sval = true;
-    const urlParams = new URLSearchParams(location.href);
-    if (urlParams.get(sname) === 'false') sval = false;
-    return sval;
+    const urlParams = new URLSearchParams(location.search);
+    if (urlParams.get(sname) === 'false') return false;
```

Comment:
Is this not simply. return urlParams.get(sname) ?

Reply:
Hello,

And sorry for the late answer.

`return urlParams.get(sname)` would return `'false'` and not a boolean (`false`)